otherlibs/action-plugin/src/protocol.ml

```diff
@@ -24,16 +14,20 @@ module Dependency = struct
           ; glob : string
           }
 
-    let sexp_of_t : _ -> Sexp.t = function
-      | File path -> List [ Atom "file"; Atom path ]
-      | Directory path -> List [ Atom "directory"; Atom path ]
-      | Glob { path; glob } -> List [ Atom "glob"; Atom path; Atom glob ]
-
-    let t_of_sexp : Sexp.t -> _ = function
-      | List [ Atom "file"; Atom path ] -> Some (File path)
-      | List [ Atom "directory"; Atom path ] -> Some (Directory path)
-      | List [ Atom "glob"; Atom path; Atom glob ] -> Some (Glob { path; glob })
-      | _ -> None
+    let conv =
+      let open Conv in
+      let file = constr "File" string (fun s -> File s) in
```

Comment:
Why did the capitalisation change? Was this intentional?

Reply:
Yes, it's the convention we use in RPC. Though if you want, I can revert this change.

Reply:
OK, fine if this is Dune RPC's convention. It seems a bit inconsistent with Dune language but it's hardly worth debating over.